smart_campus/app/models.py

```diff
@@ -181,43 +181,32 @@ def __str__(self):
         return self.name
 
 
-class QuestionType:
-    BINARY = 0x01
-    MULTIPLE_CHOICE = 0x02
+class YesNoQuestion(models.Model):
+    content = models.CharField(max_length=254)
+    answer = models.BooleanField(default=True)
 
 
 class Question(models.Model):
     content = models.CharField(max_length=254)
-
-    question_type = models.IntegerField(
-        choices=(
-            (QuestionType.BINARY, 'True or False'),
-            (QuestionType.MULTIPLE_CHOICE, 'MuitipleChoice'),
-        ),
-        default=QuestionType.MULTIPLE_CHOICE,
-    )
-    choices = models.ManyToManyField(
-        'Choice',
-        through='QuestionChoice',
+    linked_station = models.ForeignKey(
+        'Station',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL
     )
-    linked_station = models.ForeignKey('Station', null=True, on_delete=models.SET_NULL)
 
     def __repr__(self):
         return str(self.id)
 
 
-class Choice(models.Model):
-    content = models.CharField(max_length=50)
-
-    def __str__(self):
-        return self.content
-
-
 class QuestionChoice(models.Model):
```

Comment:
But the model have same style part like `StationCategory`, `StationImage` or `UserGroup`
I think it can present the relationship to the question, and maintain the code style united.

Reply:
It's just because "category," "image" and "group" are commonly used words.
It makes sense to link questions with choices without the prefix.

Reply:
Okay, I will fixed it.